### DISCUSSION TITLE
Exploring Alternative Blazor UI Toolkits Beyond Bit Blazor UI in Project Templates such as `MudBlazor`

### DISCUSSION THREAD

**Discussion by zandiarash** [bitfoundation/bitplatform#5617] [General]
Can [Mudblazor](https://mudblazor.com/) be used in BitPlatform Admin Panel ?

**Comment by yasmoradi** [comment]
Sure, why not? @zandiarash

**Reply by zandiarash** [reply]
We want to start a larg-scaled project via BitPlatform.
So you believe that there is no conflicts between them.
Thanks

**Reply by yasmoradi** [reply]
If you're interested, we can set up a free online meeting to talk about Blazor, the Bit platform, and your project. Our discussion will provide valuable insights, thanks to our experience in handling complex projects with Blazor.

Feel free to message me at `+31684207362` on WhatsApp, Telegram, or your preferred platform.

**Reply by zandiarash** [reply]
I appreciate your kindness, I am delighted to have the opportunity to connect with you.

**Comment by msynk** [comment]
but the question is why do you want to do such a thing?
Is there anything missing from bit BlazorUI that you want? if yes, please inform us about them by filing issue(s).

**Reply by zandiarash** [reply]
There are plenty of practical components accessible through MudBlazor which are missing in BitPlatform BlazorUI like **multi selection autocomplete input** and etc.
BlazorUI is awesome but not as mature as MudBlazor and has a long way to go.
By the way, thanks for the active community and spotless platform.

**Reply by msynk** [reply]
I see. Thanks.
As I've said it would be appreciated to file issues for these missing components.
I think we can add the missing features so fast that you could try to use bit BlazorUI components instead.

**Reply by zandiarash** [reply]
Sure, Please visit [#5677 ](https://github.com/bitfoundation/bitplatform/issues/5677)

**Reply by zandiarash** [reply]
> but the question is why do you want to do such a thing? Is there anything missing from bit BlazorUI that you want? if yes, please inform us about them by filing issue(s).

[Please consider the discussion #8243](https://github.com/orgs/bitfoundation/discussions/8243)
